### PR TITLE
RTL adjustments.

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
- Closes https://github.com/eventespresso/event-espresso-core/issues/2885

### CODE REVIEW

**Comment by nicolad** [comment]
image

**Comment by nicolad** [comment]
image

**Comment by nicolad** [comment]
image

**Comment by nicolad** [comment]
image

**Comment by nicolad** [comment]
image

**Comment by nicolad** [comment]
image

File: assets/src/infrastructure/ui/display/pagination/Pagination.tsx [hunk 1]
```diff
@@ -26,12 +21,12 @@ const Pagination: React.FC<PaginationProps> = ({
 	perPageOptions = ['2', '6', '12', '24', '48'],
 	showPerPageChanger,
 	total,
-	...rest
+	...props
```
Comment (tn3rb) [inline assets/src/infrastructure/ui/display/pagination/Pagination.tsx:24]: 👍

File: assets/src/infrastructure/ui/display/pagination/Pagination.tsx [hunk 2]
```diff
@@ -41,7 +36,6 @@ const Pagination: React.FC<PaginationProps> = ({
 				pageSize={perPage}
 				showSizeChanger={false}
 				total={total}
-				style={paginationStyle}
```
Comment (tn3rb) [inline assets/src/infrastructure/ui/display/pagination/Pagination.tsx:44]: # YUUUUGE 👍

File: assets/src/infrastructure/ui/display/pagination/style.scss [hunk 3]
```diff
@@ -6,6 +6,16 @@ $border-radius: var(--ee-border-radius-default);
...
 
+	ul {
+		display: flex;
+		margin-right: 1em;
+
+		[dir='rtl'] & {
+			margin-right: 0;
+			margin-left: 1em;
+		}
+	}
```
Comment (tn3rb) [inline assets/src/infrastructure/ui/display/pagination/style.scss:17]: beautiful

File: assets/src/infrastructure/ui/display/pagination/style.scss [hunk 4]
```diff
@@ -43,9 +58,9 @@ $border-radius: var(--ee-border-radius-default);
 	&__per-page {
 		height: 2.75rem;
 
-		&.ee-select {
+		&.ee-select.ee-select {
 			border: $border-style;
-			border-radius: $border-radius !important;
+			border-radius: $border-radius;
```
Comment (tn3rb) [inline assets/src/infrastructure/ui/display/pagination/style.scss:63]: 👍

File: assets/src/infrastructure/ui/layout/modal/styles.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.ee-modal {
+	&__header {
+		[dir='rtl'] & {
+			padding-right: var(--ee-padding-extreme);
+		}
+	}
+}
```
Comment (tn3rb) [inline assets/src/infrastructure/ui/layout/modal/styles.scss:7]: 👍

**Comment by manzoorwanijk** [comment]
Good job 👍

**Pull request merged**